test/parallel/test-fs-readfile-empty.js

```diff
@@ -20,27 +20,27 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
```

Comment:
The `common` module should be required in tests before any other module: https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#lines-1-3

I’m a little surprised that the linter didn’t pick this up.

Reply:
Me too... 🤔